emperor/support_files/js/sceneplotview3d.js

```diff
@@ -397,7 +397,12 @@ define([
           text = decomp.axesNames[index];
         }
       }
-      text += ' (' + decomp.percExpl[index].toPrecision(4) + ' %)';
+
+      // account for custom axes (their percentage explained will be -1 to
+      // indicate that this attribute is not meaningful).
+      if (decomp.percExpl[index] >= 0) {
```

Comment:
Is there any other way around? I think it's not really intuitive and more when you print the plot and you don't know which axis is which.